geniza/corpus/views.py

```diff
@@ -215,10 +215,8 @@ def get(self, request, *args, **kwargs):
             raise
 
     def get_solr_lastmodified_filters(self):
-        """Query solr for the object that needs their "last_modified" attribute updated.
-        Overwrites `SolrLastModifiedMixin`'s builtin method.
-        """
-        return {"pgpid_i": self.kwargs["pk"]}
+        """Filter solr last modified query by pgpid"""
+        return {"pgpid_i": self.kwargs["pk"], "item_type_s": "document"}
```

Comment:
sorry if I was unclear: we need the item type filter on the search view. (I wasn't thinking about adding it here: it may not strictly be necessary here, but it's probably a good idea)

Reply:
oh whoops, yeah, here it is [7e57d3f](https://github.com/Princeton-CDH/geniza/pull/669/commits/7e57d3fc087966cc4b8e7dfdc1d79654d1f0fef5)